src/webpack.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { IgnorePlugin } from 'webpack';
+
+import { REACT013, REACT155 } from './version';
+
+// eslint-disable-next-line import/prefer-default-export
+export function getPluginsForInstalledReact() {
```

Comment:
not really, i made it a named export to make the name of the function clarify what it does